src/views/DirectEditing.vue

```diff
@@ -95,12 +54,7 @@ export default {
 			return JSON.parse(this.initial.session) || null
 		},
 		isMobile() {
```

Comment:
```suggestion
		isMobileApp() {
```

Nitpicking, but since you named the vue method `isMobileApp()` in `Editor.vue`, let's name it this way here as well, no?